addons/product_expiry/views/production_lot_views.xml

```diff
@@ -35,4 +35,17 @@
         </field>
      </record>
 
+    <record id="view_production_lot_view_tree" model="ir.ui.view">
+        <field name="name">stock.production.lot.tree.inherit.stock</field>
```

Comment:
Guideline : as module2 = product_expiry here in name : model.view.tree.inherit.module2
this should become : `stock.production.lot.tree.inherit.product_expiry`

Same for all the other inherit here.

Reply:
@dbeguin changes  done.